pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -129,7 +133,9 @@ var _ Policy = &staticPolicy{}
 // NewStaticPolicy returns a CPU manager policy that does not change CPU
 // assignments for exclusively pinned guaranteed containers after the main
 // container process starts.
-func NewStaticPolicy(topology *topology.CPUTopology, numReservedCPUs int, reservedCPUs cpuset.CPUSet, affinity topologymanager.Store, cpuPolicyOptions map[string]string) (Policy, error) {
+func NewStaticPolicy(ctx context.Context, topology *topology.CPUTopology, numReservedCPUs int, reservedCPUs cpuset.CPUSet, affinity topologymanager.Store, cpuPolicyOptions map[string]string) (Policy, error) {
+	logger := klog.LoggerWithName(klog.FromContext(ctx), "static")
```

Comment:
Should we add the "policy" key here?
```suggestion
logger := klog.LoggerWithName(klog.FromContext(ctx), "policy", "static")
```